### DISCUSSION TITLE
Import mapping go file directly

### DISCUSSION THREAD

**Discussion by kotyara85** [oapi-codegen/oapi-codegen#1504] [Q&A]
Hello everyone,
Let's say I do have a go file with a struct I want to import map into an existing openapi spec. Is this possible to do at all besides creating a spec in that repo to describe the package structs.
Thanks

**Comment by jamietanna** [comment]
Hey, do you mean you want to refer to an external type from your OpenAPI spec? 

I.e. the `x-go-type` extension may work? https://github.com/deepmap/oapi-codegen?tab=readme-ov-file#openapi-extensions
